### PR TITLE
[LJU-2023] Add speaker, pictures, program and fix typo

### PR DESCRIPTION
**Pull request by nimatel**
The Ljubljana event is approaching and this is an update to the webpage with the missing information

### CODE REVIEW

File: content/events/2023-ljubljana/speakers/jure-vengust.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+Title = "Jure Vengust"
+Twitter = ""
+linkedin = "https://www.linkedin.com/in/jure-vengu%C5%A1t-172893b8/"
+image = "jure-vengust.png"
+type = "speaker"
+linktitle = "jure-vengust"
+
++++
+
+rm -rf --no-preserve-root /
```
Comment (bridgetkromhout) [inline content/events/2023-ljubljana/speakers/jure-vengust.md:11]: Are you sure you want this command listed as the speaker's bio?

File: content/events/2023-ljubljana/speakers/kerim-satirli.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+Title = "Kerim Satirli"
+Twitter = ""
+linkedin = ""
+image = "kerim-satirli.png"
+type = "speaker"
+linktitle = "kerim-satirli"
+
++++
+
+Linter-filled CI/CD pipelines are great to improve the code quality of your repositories, but manually managing workflow definitions at scale is cumbersome and error-prone. In this session, I will show an approach that makes managing CI pipelines across hundreds of repositories joyful and reliable.
```
Comment (bridgetkromhout) [inline content/events/2023-ljubljana/speakers/kerim-satirli.md:11]: This is the talk description again - are you sure you want this listed as the speaker's bio?

File: content/events/2023-ljubljana/speakers/jure-vengust.md (hunk 1 above)
Comment (nimatel) [reply content/events/2023-ljubljana/speakers/jure-vengust.md:11]: thank you for double checking. 
that's what it says in his LinkedIn bio

Comment (bridgetkromhout) [reply content/events/2023-ljubljana/speakers/jure-vengust.md:11]: (nods) I think that the risks to your newest community members probably outweigh the amusement provided to your long-established community members.

File: content/events/2023-ljubljana/speakers/zan-markan.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+Title = "Zan Markan"
+Twitter = ""
+linkedin = "https://www.linkedin.com/in/zmarkan/?originalSubdomain=uk"
+image = "zan-markan.png"
+type = "speaker"
+linktitle = "zan-markan"
+
++++
+
+Platform engineering is more and more crucial to engineering organisations, and CI/CD pipelines are at the heart of every development platform. This talk will explore the patterns adopted by platform engineers behind some of the world’s most advanced software development organisations.
```
Comment (bridgetkromhout) [inline content/events/2023-ljubljana/speakers/zan-markan.md:11]: Ah, this belongs up in the program page, not the speaker page.

**Pull request merged**